### ISSUE TITLE
Close (x) box on errors not accessible via keyboard

### ISSUE THREAD

**Issue by andrebtlima** [CityOfBoston/Iterators#115]
https://bostond8dev2.prod.acquia-sites.com/45-lothrop-street-beverly
 
Type of User:  Anonymous
 
 
Environment
(Device > OS > Browser)
Computer > macOS > Safari
 
Brief Description
Some elements not accessible with the keyboard.

Steps to reproduce
00:00, Click in the URL of the browser and then start using the Tab key in order to navigate the interface using only the keyboard.

00:10, Tab into the side menu.  Note that bugs related to this are already reported as part of the boston.gov project.
   a.     00:12, Use the mouse to click out of the side menu in order to workaround this bug.
   b.     Continue to tab with the keyboard.

 00:16, Notice that the search icon is not accessible with the keyboard.  This is already reported as part of the boston.gov project.

00:23, Tab into the Google Map.  Note that this component will be tested for accessibility separately.  A quick Internet search indicates that this module has some accessibility issues.  Further research will be done.

00:25, Tab to the email button, and the tab to the Print button
    a.     BUG:  The focus is not consistent on these two options.  One is highlighted in blue and the other changes color to red.

00:27, Tab past the Print button
     a.     BUG:  The focus moves to “Calculate your eligibility” and past the 
     expansion triangle for the Unit information. Therefore, it is not possible to 
     expand and contract the unit information using only the keyboard.

00:31, Tab to the tiles in the “Learn about affordable housing” section.
     a.     Bug:  It is difficult to see the focus of the element.  Also, the focus is done 
     differently with a mouse hover as compared to using the keyboard.  This is 
     already reported as part of the boston.gov project.

00:35, Continue to tab through the user interface and the user is able to successfully cycle back to the beginning of the interface.
 
 
Desired Result
Ability to navigate the entire user interface with the keyboard.
 
Actual Result
Some elements cannot be reached and some focus issues observed.

**Comment by subaha-cob** [comment]
To test go to:
https://d8-uat.boston.gov/metrolist/search/housing/90-new-england-ave?type=sale&assignment=lottery

**Comment by subaha-cob** [comment]
Testing at: https://d8-uat.boston.gov/

**Comment by david-iterators** [comment]
Verified using https://d8-uat.boston.gov/metrolist/search/housing/lakeview-estates?type=sale&assignment=lottery#print
